src/Containers/Microsoft.NET.Build.Containers/Microsoft.NET.Build.Containers.csproj

```diff
@@ -48,7 +48,7 @@
     <PackageReference Include="System.Text.Json" ExcludeAssets="runtime">
       <!-- For the net472 flavor, lock back to a System.Text.Json equal to or older than MSBuild will provide
            so it doesn't have to be redistributed. -->
-      <VersionOverride Condition="'$(TargetFramework)' == 'net472'">8.0.3</VersionOverride>
+      <VersionOverride Condition="'$(TargetFramework)' == 'net472'">$(SystemTextJsonToolsetPackageVersion)</VersionOverride>
```

Comment:
@baronfel any concerns about the package version of containers moving its lowest-supported-msbuild up?

Reply:
Only about VS compatibility, which you have a pretty good handle on. Theory question - this toolset package version is used by all SDK tooling inserted into/used by VS, right? If so, we'd be no more broken here than the rest of the stack if the STJ version advances beyond the supported VS pathway?

And the long-term fix for this is to netcore taskhost the world so that we aren't beholden to VS library constraints?

Reply:
> Theory question - this toolset package version is used by all SDK tooling inserted into/used by VS, right? If so, we'd be no more broken here than the rest of the stack if the STJ version advances beyond the supported VS pathway?

Yes, the only distinction is using a new version of the containers _package_ on an older SDK/VS. Is that still supported?

> And the long-term fix for this is to netcore taskhost the world so that we aren't beholden to VS library constraints?

correctamundo

Reply:
>  new version of the containers package on an older SDK/VS. Is that still supported?

I think we can take this because:
* the main feature we're working on for containers (multi arch) will be landing in 8.0.4xx packages and will not have this change, and
* we'll likely push for netcore taskhost in 10.x so we can remove the net framework version of this anyway